package.yml

```diff
@@ -25,4 +25,11 @@ pages:
 
 requires:
     redaxo: '^5.0'
-    php: '>=5.5'
\ No newline at end of file
+    php: '>=5.5'
+
+conflicts:
```

Comment:
Vllt solltest du hier nen kommentar reinmachen was diese regel bewirkt... conflict rules sind bisher in FOR nicht üblich